src/build/app/mod.rs

```diff
@@ -1200,23 +1307,107 @@ impl<'help> App<'help> {
         self
     }
 
-    /// Replaces an argument to this application with other arguments.
+    /// Replaces an argument or subcommand used on the CLI at runtime with other arguments or subcommands.
     ///
-    /// Below, when the given args are `app install`, they will be changed to `app module install`.
+    /// When this method is used, `name` is removed from the CLI, and `target`
+    /// is inserted in it's place. Parsing continues as if the user typed
+    /// `target` instead of `name`.
+    ///
+    /// This can be used to create "shortcuts" for subcommands, or if a
+    /// particular argument has the semantic meaning of several other specific
+    /// arguments and values.
+    ///
+    /// Some examples may help to clear this up.
     ///
     /// # Examples
     ///
+    /// We'll start with the "subcommand short" example. In this example, let's
+    /// assume we have a program with a subcommand `module` which can be invoked
+    /// via `app module`. Now let's also assume `module` also has a subcommand
+    /// called `install` which can be invoked `app module install`. If for some
+    /// reason, users needed to be able to reach `app module install` via the
+    /// short-hand `app install` we have several options.
```

Comment:
```suggestion
    /// reason users needed to be able to reach `app module install` via the
    /// short-hand `app install`, we'd have several options.
```

Reply:
We don't use this style either.

Reply:
OK.

Could you please be more specific about these "wrong style" rejections? I don't think we have any officially (or even de-facto) established style for our documentation, and I don't think we'd be able to maintain it if we had. At any rate, this particular suggestion has _nil_ to do with style, it's a grammatical correction. Let me spell it out.

Let's look at the whole sentence:

> If for some reason users **needed** to be able to reach `app module install` via the
short-hand `app install`, we **have** several options.

What tense is the conditional verb in (to need)? Past tense, therefore this is [Second Conditional](https://www.ef.com/ca/english-resources/english-grammar/conditional/) clause (or Type 2, in some sources). Now, which form the main verb (to have) should use in Second Conditional? Present conditional, which is "would have". Which form it _is_ written in? "have". This is a grammatical error. Now let's recall that "we'd" is a common shortcut for "we would", and everything is in place. Class, lesson over.

Sidenote: I think using First conditional would fit better here, but Kevin chose Second for some reason. Anyway, it's not wrong. I would also replace "we" with "they".

So what was this "style" crap I was called on? Am I being viewed through some weird lenses that give some extra wight to the "he's is prone to sneak some expletives and undue jokes in documentation" hypothesis? How long do I need to pay for my past mistakes; is several months without incident and with a lot of kudos from users not enough?

Reply:
"we'd have several options" makes it **passive voice**. Common convention for documentation is to use **active voice**. It should be "there are several options".

Reply:
Passive voice would be, "several options would be had [by us]" (forming passive voice with Second Conditional + "have" is kinda really clumsy). The "would" doesn't magically make it passive voice. I'm not sure what rules you are being guided by - and I would very much like a reference here so I knew which books to avoid - but you are just being wrong here.

If it helps, passive voice always implies use of "to be" in one of its forms (there are [some exceptions](https://www.quora.com/Can-we-use-the-passive-voice-without-the-verb-to-be), but they are all about "to be is replaced by get, went, had and such"; none of them apply here). Do you spot it in the sentence? Hint: you do, but not in the place you pointed at, and it doesn't make the second part of the sentence passive.